moodle/pkg/apis/moodlecontroller/v1/types.go

```diff
@@ -28,19 +28,19 @@ type MoodleSpec struct {
 	//MySQL Username
 	MySQLUserName string `json:"mySQLUserName"`
 	//MySQL Password
-	MySQLUserPassword string `json:"mySQLUserPassword"`
+	MySQLPassword string `json:"mySQLPassword"`
```

Comment:
Don't need to change this attribute name. We can keep it as MySQLUserPassword.

Reply:
Ok